prow/plank/README.md

```diff
@@ -0,0 +1,40 @@
+# Plank
+
+Plank is the controller that manages the job execution and lifecycle for jobs running in k8s.
+
+### Usage
+```bash
+$ bazel run //prow/cmd/plank -- --help
+```
+
+### Configuration
+Only GCS is supported as the log storage at the moment.
```

Comment:
nit: Please clarify this to "job log storage", otherwise this could be understood as referring to Plank's logs.

Reply:
yeah - technically the artifacts is not handled by plank